lib/network/NetworkUtil.js

```diff
@@ -1,20 +1,21 @@
 let util = require("../util");
 
 /**
- * @class NetworkUtil
+ * Utility Class
  */
 class NetworkUtil {
   /**
-   * @constructor NetworkUtil
+   * @inheritDoc
```

Comment:
Uhm, not lower-case `@inheritdoc`?
Also, there's no parent class to inherit from, so what's the use of this tag?

Reply:
Will change casing (although this casing does work.)
Inherits doc from encapsulating class.

Reply:
[From my understanding](http://usejsdoc.org/tags-inheritdoc.html) the doc is derived from the parent class, which is not the same as the encompassing class. I think the tag is misplaced here.

Reply:
Will change to `@ignore`, since constructors are better documented in their class definition.